moonspeak/frequency/frequencyapp/utils.py

```diff
@@ -10,78 +10,82 @@
 import filetype
 from collections import Counter
 import os
-from .models import RequestCounter
+from .models import RequestCounter, Task
 
 japan_ords = set(i for i in range(19969, 40959))
 
 
-def frequency(user_string):
+def frequency(user_string: str) -> dict:
+    """Checks whether a symbol is a Japanese character"""
     return {k: v for k, v in Counter(user_string).most_common() if ord(k) in japan_ords}
```

Comment:
кстати, кстати, предлагаю все-таки поставить верхную границу типо 1000 ироглифов?

```suggestion
    return {k: v for k, v in Counter(user_string).most_common(1000) if ord(k) in japan_ords}
```